controllers/AuthController.js

```diff
@@ -47,18 +50,26 @@ class AuthController {
     const userRecord = await userModelManager.findUser('email', req.body.email);
     if (userRecord) {
       const { id, password, isverified } = userRecord.dataValues;
-
+      const userRole = await getUserRole(id);
       const decryptedPassword = bcrypt.compareSync(req.body.password, password);
       if (decryptedPassword) {
         verificationStatus = checkVerificationStatus(isverified);
-
         if (verificationStatus) {
-          const token = jwtUtil.generateToken(TOKEN_TIMESPAN, id);
-          return res.status(OK_CODE).json({
-            success: SUCCESSFUL,
-            message: 'Login Successful',
-            loginToken: token
-          });
+          let token;
+          const payLoad = {
+            id
+          };
+          if (!userRole[0]) {
+            token = jwtUtil.generateToken(TOKEN_TIMESPAN, id);
+            return res.status(OK_CODE).json({
+              success: true,
+              message: SUCCESS_LOGIN,
+              loginToken: token
+            });
+          }
+          payLoad.status = userRole[0].dataValues.role;
```

Comment:
` payLoad.status = userRole[0].dataValues.role;` 

Good job so far but I would love you to answer the following questions;

What happens when a user has more than one role in the system?
What also happens in a scenario where a different endpoint should only be accessed by users with a certain role other than the admin role?

Please read through the story description again
https://www.pivotaltracker.com/story/show/162414182

Reply:
I have fixed it, A user can have many roles and the role is automatically assigned to the user payload on login